amy/scripts/seed_emails.py

```diff
@@ -0,0 +1,61 @@
+import logging
+from typing import TypedDict
+from uuid import UUID
+
+from emails.models import EmailTemplate
+from workshops.utils.seeding import deprecate_models, seed_models
+
+logger = logging.getLogger("amy")
+
+# If an entry needs to be removed from the database, remove it from e.g.
+# `EMAIL_TEMPLATES`, and put its' ID in `DEPRECATED_EMAIL_TEMPLATES`.
+
+DEPRECATED_EMAIL_TEMPLATES: list[str] = []
+
+EmailTemplateDef = TypedDict(
+    "EmailTemplateDef",
+    {
+        "active": bool,
+        "id": UUID,
+        "name": str,
+        "signal": str,
+        "from_header": str,
+        "reply_to_header": str,
+        "cc_header": list[str],
+        "bcc_header": list[str],
+        "subject": str,
+        "body": str,
+    },
+)
+
+EMAIL_TEMPLATES: list[EmailTemplateDef] = [
+    EmailTemplateDef(
+        active=True,
+        id=UUID("5703079f-2f37-4aee-8267-fe6db3eee870"),
+        name="Person records are merged",
+        signal="persons_merged",
```

Comment:
No, it's `persons_merged` (see https://github.com/carpentries/amy/pull/2434/files#diff-fa5a5ef6a674471029d02096c7f7694f7f106b89fa3f175bc5081bb51cf3b3feR33). I can change it to the same name as instance of `Signal` class, which is defined as `persons_merged_signal` in https://github.com/carpentries/amy/pull/2434/files#diff-770ab91787786daf35b394421418fd5b7cdcbc526357f6072a098ae9208b7cf0R8, if you think it would be clearer.

Reply:
I think it's fine for them to be different. I didn't realise that these are sort of separate things - a string that is used as a key to get the relevant EmailTemplate, and an actual `Signal`, which are only connected by the receiver function.

It would be a bit easier to follow if there was some sort of `Signal.name` attribute, that could be passed through to the email template instead.

Reply:
Unfortunately there's no such thing as `Signal.name`, I checked. I may add a custom field, or perhaps subclass `Signal` to achieve this result.